### ISSUE TITLE
`text-transform` interpreted as `transform2d`

### ISSUE THREAD

**Issue by abea** [stylelint/stylelint#2065]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug where `text-transform` is throwing a linter error related to `transform2d`.

> Which rule, if any, is this issue related to?

`no-unsupported-browser-features`

> What CSS is needed to reproduce this issue?

e.g.

```css
.nav {
  text-transform: uppercase;
}
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    "no-unsupported-browser-features": true
  }
}
```

> Which version of stylelint are you using?

7.5.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

through VS Code plugin vscode-stylelint

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Occurring in SCSS files.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

My file is throwing an error on the text-transform line, `severity: 'Error' message: 'stylelint: Unexpected browser feature "transforms2d" is not supported by Opera Mini all (no-unsupported-browser-features)'`.

**Comment by evilebottnawi** [comment]
@abea you should report to https://github.com/anandthakker/doiuse repo, this rule works based on this package

**Comment by abea** [comment]
Thanks. It looks like it's been reported to them without action in a while. https://github.com/anandthakker/doiuse/issues/8
